hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java

```diff
@@ -468,9 +468,14 @@ private boolean deleteAction(Action<Boolean> deletion, String type, Path dir) {
       LOG.debug("Couldn't delete '{}' yet because it isn't empty w/exception.", dir, exception);
       deleted = false;
     } catch (IOException ioe) {
-      LOG.info("Could not delete {} under {}. might be transient; we'll retry. if it keeps "
-          + "happening, use following exception when asking on mailing list.",
-        type, dir, ioe);
+      if (LOG.isTraceEnabled()) {
+        LOG.info("Could not delete {} under {}; will retry. If it keeps happening, " +
+            "quote the exception when asking on mailing list.", type, dir, ioe);
+      } else {
+        LOG.info("Could not delete {} under {}; will retry. If it keeps happening, enable" +
+            "TRACE-level logging and quote the exception when asking on mailing list.",
+            type, dir, ioe.getMessage());
```

Comment:
nit: space between `enable` and `TRACE-level`. 

One more concern. I may be wrong in this.` ioe.getMessage()` will be treated as string and not  Throwable. Since we are using string formatter, we need another `{}` formatter to add exception's message to log. 
@saintstack  Could you please verify that log is printed as expected ? Thank you !

Reply:
This is a good one. Thank you. Fixed.